scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -841,7 +850,11 @@ def __init__(self, test_file, pics_file, definitions):
         self.name = _value_or_none(data, 'name')
         self.PICS = _value_or_none(data, 'PICS')
 
-        self._parsing_config_variable_storage = _value_or_none(data, 'config')
+        self._parsing_config_variable_storage = data.get('config', {})
+        self.__populate_default_config_if_missing('nodeId', 0x12345)
```

Comment:
Where is this default coming from?

Reply:
All these defaults are from:
`third_party/zap/repo/dist/src-electron/generator/matter/app/zap-templates/common/variables/Variables.js`

Reply:
Thanks!